tests/data/test_data_preprocessing_utils.py

```diff
@@ -156,7 +161,10 @@ def test_load_dataset_with_hf_dataset(hf_dataset, splitName):
         processor_config=DataPreProcessorConfig(), tokenizer=None
     )
     load_dataset = processor.load_dataset(
-        datasetconfig=datasetconfig, splitName=splitName, datafile=None
+        datasetconfig=datasetconfig,
+        streaming=processor.processor_config.streaming,
+        splitName=splitName,
+        datafile=None,
     )
     assert isinstance(load_dataset, Dataset)
```

Comment:
could there be a case that this doesn't hold? 
can we make this assert more explicity 

```
assert processor.processor_config.streaming is False and isinstance(load_dataset, Dataset)
```